docs/fact-token/utility-token.md

```diff
@@ -100,7 +101,8 @@ important parameters that define the estimated rewards (e.g. number of feeds).
 The yellow fields can be modified by the user. Some fields include notes to help
 users understand how the calculation works.
 
-[calc-1]: https://docs.google.com/spreadsheets/d/1aH4Zwtn8KUTtrdzBBZFK1_Kulb7_a4uJGjzLhmPTLFc/edit#gid=1934045699
+[calc-1]:
+    https://docs.google.com/spreadsheets/d/1aH4Zwtn8KUTtrdzBBZFK1_Kulb7_a4uJGjzLhmPTLFc/edit#gid=1934045699
```

Comment:
again, this just looks wrong. If it causes no issues disregard

Reply:
It looks weird in the Git diff viewer, but actually it's just indented. The way Prettier formats it is if the entire url and title can fit within 80 characters, it will put it on 1 line but if not, it breaks the url portion to a second line and indents that part. See below 👇

<img width="678" alt="Screenshot 2024-11-12 at 4 52 52 PM" src="https://github.com/user-attachments/assets/5226710a-c507-4562-ad8b-1229954ed3fb">